Documentation/install/system_requirements.rst

```diff
@@ -151,22 +151,25 @@ by adding the following to the helm configuration command line:
 
 .. _features_kernel_matrix:
 
-Advanced Features and Required Kernel Version
-=============================================
-Cilium requires Linux kernel 4.9.17 or higher, however development on additional
-kernel features and functionality continues to progress in the Linux community.
-Some Cilium features and functionality are dependent on newer kernel versions.
-These additional Cilium features and functionality are enabled by upgrading to
-a later kernel version as detailed below:
-
-======================= ===============================
-Cilium Feature          Minimum Kernel Version
-======================= ===============================
-IPv4 fragment tracking  >= 4.10
-:ref:`cidr_limitations` >= 4.11
-:ref:`host-services`    >= 4.19.57, >= 5.1.16,  >= 5.2
-:ref:`kubeproxy-free`   >= 4.19.57, >= 5.1.16,  >= 5.2
-======================= ===============================
+Required Kernel Versions for Advanced Features
+==============================================
+
+Cilium requires Linux kernel 4.9.17 or higher; however, development on
+additional kernel features continues to progress in the Linux community. Some
+of Cilium's features are dependent on newer kernel versions and are thus
+enabled by upgrading to more recent kernel versions as detailed below.
+
+============================= ===============================
+Cilium Feature                Minimum Kernel Version
+============================= ===============================
+:ref:`concepts_fragmentation` >= 4.10
+:ref:`cidr_limitations`       >= 4.11
+:ref:`host-services`          >= 4.19.57, >= 5.1.16,  >= 5.2
+:ref:`kubeproxy-free`         >= 4.19.57, >= 5.1.16,  >= 5.2
+:ref:`bandwidth-manager`      >= 5.1
+:ref:`session-affinity`       >= 5.7
```

Comment:
We do have a fallback mode for the session-affinity when running on <5.7 kernel:
> The session affinity support is enabled by default for Cilium’s kube-proxy replacement. For users who run on older kernels which do not support the network namespace cookies, a fallback in-cluster mode is implemented, which is based on a fixed cookie value as a trade-off. This makes all applications on the host to select the same service endpoint for a given service with session affinity configured. To disable the feature, set config.sessionAffinity=false.

https://docs.cilium.io/en/v1.8/gettingstarted/kubeproxy-free/#session-affinity

Reply:
So the downside for users of not having network namespace cookies is that all pods on a given node will be serviced by the same backend?

That's a bit hard to convey in a single phrase for the system requirements list. Maybe we could say "Per-namespace session affinity >= 5.7"? Or just fallback to "Full support for session affinity >= 5.7"?

Reply:
> So the downside for users of not having network namespace cookies is that all pods on a given node will be serviced by the same backend?

That's correct, but only for E-W traffic.

> Maybe we could say "Per-namespace session affinity >= 5.7"? Or just fallback to "Full support for session affinity >= 5.7"?

Hmm, `per-namespace session affinity` might confuse users, so maybe just `full support for session affinity`?

Reply:
Please see #13318.

Reply:
Thanks!